ruby-otlp/app.rb

```diff
@@ -1,67 +1,22 @@
-require 'sinatra/base'
 require 'opentelemetry/sdk'
 require 'opentelemetry/exporter/otlp'
+require 'rubygems'
+require 'bundler/setup'
 
+Bundler.require
 
-# configure SDK with OTLP exporter
 OpenTelemetry::SDK.configure do |c|
+  c.service_name = 'ruby-otlp'
+  c.use 'OpenTelemetry::Instrumentation::Sinatra'
   c.add_span_processor(
     OpenTelemetry::SDK::Trace::Export::BatchSpanProcessor.new(
-      exporter: OpenTelemetry::Exporter::OTLP::Exporter.new(
-		endpoint: 'localhost:55681/v1/trace', # send to local collector
-		insecure: true
+      OpenTelemetry::Exporter::OTLP::Exporter.new(
+        endpoint: 'http://localhost:55681/v1/traces'
```

Comment:
I recommend omitting this custom config in code and adding to the README a step to set the `OTEL_EXPORTER_OTLP_ENDPOINT` environment variable to `http://localhost:55681`. That will demonstrate configuration through env vars instead of code changes. (Also `/v1/traces` was recently declared as assumed in the spec and Ruby OTel SDK has caught up to that.)

I'm happy to make these edits, too, if you like.

Reply:
Updated to include extra instructions to set env var in README and updated URL to omit the `/v1/traces`